### ISSUE TITLE
Implement iPython like notebook

### ISSUE THREAD

**Issue by ramnathv** [ramnathv/rNotebook#9]
The idea is to have a single column document with editable code cells, just like an iPython notebook. It should allow users to execute a single code chunk or the entire document.

I think this should not be very complicated. In fact, I already have an interactive single column version of the console, with output below, and I should be able to tweak it.

**Comment by karthik** [comment]
Would also be nice to have output cached so a document can be viewed through something like nbviewer.

**Comment by ramnathv** [comment]
Yes, that is what I was thinking. So pretty much, just run like you would run `knit2html`, but modify the default markup, that makes it easy to use javascript to convert the code cells to interactive, and also be able to replace outputs.
